huaweicloud/config/logger.go

```diff
@@ -25,7 +25,6 @@ var maxTimeout = 10 * time.Minute
 // customize the default http client RoundTripper to allow for logging.
 type LogRoundTripper struct {
 	Rt         http.RoundTripper
-	OsDebug    bool
```

Comment:
joined-operation cloud providers used this filed, so can we keep it?

Reply:
the joined-operation cloud providers also have the log level setting problem, I think it better to fix it in next reference